packages/node/package.json

```diff
@@ -56,6 +56,7 @@
     "content-type": "1.0.4",
     "cookie": "0.4.0",
     "etag": "1.8.1",
+    "path-to-regexp": "6.2.1",
```

Comment:
I wonder if we should update `@vercel/routing-tools` to match this latest version too (currently 6.1.0)